api-cars/index.ts

```diff
@@ -4,6 +4,7 @@ import fs from "fs"
 import { carsRouter } from "./routes/cars";
 import { usersRouter } from "./routes/users"
 const app = express();
+app.use(express.json())
```

Comment:
we added middelware for json parse - from know on all post request with body will be parsed and available in the entry point